scripts/docker/entrypoint_ci.sh

```diff
@@ -205,6 +205,8 @@ function determine_airflow_to_use() {
         mkdir -p "${AIRFLOW_SOURCES}"/tmp/
     else
         python "${IN_CONTAINER_DIR}/install_airflow_and_providers.py"
+        # Some packages might leave legacy typing module which causes test issues
+        pip uninstall -y typing || true
```

Comment:
Yep, I've also recently found this one, and this comes from `yandex-query-client`. I've asked @uzhastik to remove this one from package dependencies since it make only sense in Python < 3.5.

Reply:
Ah yeah. That would explain why it happens indeed.